aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -360,6 +375,45 @@ module std::features {
         });
     }
 
+    /// Enable and disable features *for the next epoch*.
+    ///
+    /// NOTE: when it takes effects depend on feature `RECONFIGURE_WITH_DKG`.
+    /// See `aptos_framework::aptos_governance::reconfigure()` for more details.
+    ///
+    /// Can only be called by a signer of @std.
+    public fun change_feature_flags_for_next_epoch(framework: &signer, enable: vector<u64>, disable: vector<u64>) acquires PendingFeatures, Features {
```

Comment:
How about making this always-on and ensure buffer is cleared at the end of the reconfig, regardless of the reconfig type?

Anything calling `set_x_immediately` should eventually be replaced by `set_x_for_next_epoch() + aptos_governance::reconfigure()`. There's no need to have 2 sets of API.

But since mirgating all `set_x_immediately` caller takes time, we can leave `set_x_immediately` open for now (so tests are passing).